chapter06/kds/ex3.pl

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/perl
+
+use warnings;
+use strict;
+
+print "Printing ENV hash:\n";
+
+printf "%-" . column_size() . "s => %s\n", $_, $ENV{$_} foreach ( sort keys %ENV );
```

Comment:
Nice use of sub method here, but I think you are looping for the max column width everytime in the ENV loop, actually storing the max column width in a variable will be much more efficient.